src/woo_publications/publications/admin.py

```diff
@@ -311,7 +311,7 @@ def save_related(
         change: bool,
     ):
         super().save_related(request, form, formsets, change)
-        if not change:
+        if not change or change and "informatie_categorieen" in form.changed_data:
```

Comment:
```suggestion
        if not change or (change and "informatie_categorieen" in form.changed_data):
```
for readability, can you make the precedence a bit more clear?